### ISSUE TITLE
The behavior of repr is different on different platforms

### ISSUE THREAD

**Issue by laike9m** [alexmojaki/cheap_repr#13]
Here's my program:

```python
from cheap_repr import cheap_repr

def this_is_my_function():
    def bar():
        pass

    print(cheap_repr(bar))


this_is_my_function()
```

On Mac, the result is `<function this_is_my_function.<locals>.bar at 0x1104218b0>`.

On Linux, it's `<function this_is_my_functio...ocals>.bar at 0x7f7239c9baf0>`

It seems Linux has a lower threshold for string length, but since this is not documented, I would expect the behavior to be the same across different platforms.

**Comment by alexmojaki** [comment]
The problem is just that the address at the end is a different length accross your machines:

    0x7f7239c9baf0
    0x1104218b0

Functions don't have a special repr so they fall back to the default which may be truncated. That's silly since they never get really long, so I'm making it use the normal untruncated repr.

**Comment by laike9m** [comment]
Thanks for the quick fix!

**Comment by laike9m** [comment]
Will you release a new version?

**Comment by alexmojaki** [comment]
Released 0.4.3
